cartographer/mapping/2d/grid_2d.cc

```diff
@@ -60,8 +60,9 @@ void Grid2D::SetCorrespondenceCost(const Eigen::Array2i& cell_index,
 
 // Returns the probability of the cell with 'cell_index'.
 float Grid2D::GetCorrespondenceCost(const Eigen::Array2i& cell_index) const {
-  LOG(FATAL) << "Not implemented";
-  return kUnknownCorrespondenceValue;
+  if (!limits().Contains(cell_index)) return kMaxCorrespondenceCost;
+  return ValueToCorrespondenceCost(
```

Comment:
Can you please post the performance evaluation results to make sure we do not have a performance regression?